### DISCUSSION TITLE
Architecture Help/Thoughts: Multi-site org with packet brokers

### DISCUSSION THREAD

**Discussion by ceafin** [Security-Onion-Solutions/securityonion#5760] [Unsupported Versions] [answered]
I was trying to see if anyone else has a widely distributed infrastructure using the new SO, specifically using AWS as some part of it.  I was thinking this ought to be feasible, but I'm not sure how.

Either, I would want an organization with multiple geographical sites and networks:
- to have each site their own SO on-site feeding into and/or being managed from an AWS-centralized SO, --OR--
- to have each site utilize their packet brokers to tunnel their packets up into their AWS-centralized packet broker visibility platform, and then from there ship off the appropriate streams to a completely cloud hosted SO deployment.

Either of these sound sane? Are there any hybrid architecture examples out there people have to share?

**Comment by dougburks** [comment, marked as answer]
You can probably make either architecture work, but I suspect that most organizations would go with an on-prem Security Onion forward node (sensor) that stores full packet capture locally but forwards its alerts and metadata to the Security Onion backend in AWS.

If you haven't already, make sure you check out our AWS AMI documentation:
https://docs.securityonion.net/en/2.3/cloud-ami.html
